src/sagemaker/config/config_schema.py

```diff
@@ -298,6 +298,11 @@ def _simple_path(*args: str):
 TRAINING_JOB_INTER_CONTAINER_ENCRYPTION_PATH = _simple_path(
     SAGEMAKER, TRAINING_JOB, ENABLE_INTER_CONTAINER_TRAFFIC_ENCRYPTION
 )
+SESSION_S3_BUCKET_PATH = _simple_path(SAGEMAKER, PYTHON_SDK, MODULES, SESSION, S3_BUCKET)
+SESSION_S3_OBJECT_KEY_PREFIX_PATH = _simple_path(
```

Comment:
We should add doc-string's on what these mean

Reply:
(discussed offline -- these are not surfaced to the SDK user so dont need doc strings)